en/docs/setup/data-purging.md

```diff
@@ -1,14 +1,14 @@
 # Data Purging
 
-You can perform data purging by clearing the session data using the script given below. WSO2 Identity Server stores session data for authentication purposes. As the volume of the data stored grows over
-time, the authentication operations may also eventually consume more time. You can apply data purging to mitigate this. 
+You can perform data purging by clearing the session data using the script given below. WSO2 Identity Server stores session data for authentication purposes. As the volume of the data stored grows over time, the authentication operations may also eventually consume more time. It is highly recommended to perform data purging on production servers to mitigate this.
 
 !!! info 
     For more information about session persistence, see [Authentication Session Persistence](../../learn/authentication-session-persistence).
 
 !!! tip
-    It is safe to run these steps in read-only mode or during a time when traffic on the server is low.
-    
+    It is recommended to run these steps at the time where server traffic is low. Especially if you are running this in the production environment at the first time, since the data voulue to be purged may be at a higher side. However consider this as a housekeeping task and needs to be run at regular intervals. 
```

Comment:
data volume

However, consider this as a housekeeping task that needs to be run...